doc/classes/EditorImportPlugin.xml

```diff
@@ -15,12 +60,14 @@
 			<argument index="0" name="preset" type="int">
 			</argument>
 			<description>
+				Get the options and default values for the preset at this index. Returns an Array of Dictionaries with the following keys: "name", "default_value", "property_hint" (optional), "hint_string" (optional), "usage" (optional).
 			</description>
 		</method>
 		<method name="get_importer_name" qualifiers="virtual">
 			<return type="String">
 			</return>
 			<description>
+				Get the unique name of the importer.
```

Comment:
No need to document get/set methods: document the <member> tags instead. get/setters will disappear in a future version of the docs (after we moved all existing descriptions to members)

Reply:
Can you clarify what you mean by documenting the tag?

Reply:
The member tags - turns out github treats \<member\> as an empty url so it hid it ^^

Reply:
Actually as far as I can see there are no `<member>`s tag in this document...